src/components/LightSwitch.js

```diff
@@ -1,9 +1,31 @@
-import React from "react"
+import React, { useState } from "react"
+
+const LightSwitch = (props) => {
+  const [lightSwitchStatus, setLightSwitchStatus] = useState("off")
+  const [lightBulbColor, setLightBulbColor] = useState("white")
+
+  const handleClick = () => {
+    if (lightSwitchStatus === "off") {
+      setLightSwitchStatus("on")
+      setLightBulbColor("yellow")
+    } else {
+      setLightSwitchStatus("off")
+      setLightBulbColor("white") 
+    }
+  }
 
-const LightSwitch = () => {
   return (
     <>
-      <div className="lightbulb"></div>
+      <button 
```

Comment:
I like that you made a separate button for your switch...I put my button inside of line 6 and I think it's going to cause problems. So good job!